src/Infrastructure/MembershipApplicationTrackingInfo.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Infrastructure;
+
+/**
+ * @licence GNU GPL v2+
+ * @author Jeroen De Dauw < jeroendedauw@gmail.com >
+ */
+class MembershipApplicationTrackingInfo {
+
+	private $totalImpressionCount;
+	private $singleBannerImpressionCount;
+	private $confirmationPage;
+	private $confirmationPageCampaign;
```

Comment:
I'm actually quite fuzzy about all four of these fields, so need some explanation to make sure they have sane names